py_cui_2048/__init__.py

```diff
@@ -218,11 +221,10 @@ def undo_move(self, turn_up_down):
             self.prev_score = temp_score
             self.apply_board_state()
         else:
-            self.master.show_error_popup('Error', 'No move has been made!')
-
+            self.master.show_error_popup("Error", "No move has been made!")
 
     def get_logo_text(self):
-        """Function tath gets ascii art 2048 logo
+        """Function that gets ascii art 2048 logo
 
         Returns
         -------
```

Comment:
i get a lot of warnings for this. mostly has to do with the use of escape characters `\ `

Reply:
Which formatter are you using that's giving you the warnings?

Reply:
```cmd
__init__.py:237: SyntaxWarning: invalid escape sequence '\|'
  out = out + "/ __  \|  _  |  /   ||  _  |\n"
__init__.py:238: SyntaxWarning: invalid escape sequence '\ '
  out = out + "`' / /'| |/' | / /| | \ V / \n"
__init__.py:239: SyntaxWarning: invalid escape sequence '\ '
  out = out + "  / /  |  /| |/ /_| | / _ \ \n"
__init__.py:240: SyntaxWarning: invalid escape sequence '\ '
  out = out + "./ /___\ |_/ /\___  || |_| |\n"
__init__.py:241: SyntaxWarning: invalid escape sequence '\_'
  out = out + "\_____/ \___/     |_/\_____/\n"
```

My environment is python `v3.12.1`

I have not used any of my linters and formatters on your code yet.

i mainly use **Black and Pylint**

Reply:
We can probably get around these warnings by using a raw string.

Reply:
Yes. I realised py_cui reads the ascii characters directly. i thought escaping those characters might work, but i think it may result in unintended results. I can tackle that once i am able to run the entire program in my environment. Using raw string should work